pages/Ore-Washer.md

```diff
@@ -0,0 +1,17 @@
+# Ore Washer
```

Comment:
Wiki pages should not include headers.
They have headers automatically, this would create two.